web/src/message_view_header.js

```diff
@@ -10,6 +10,14 @@ import * as recent_view_util from "./recent_view_util";
 import * as rendered_markdown from "./rendered_markdown";
 import * as search from "./search";
 
+export function get_formatted_sub_count(sub_count) {
```

Comment:
It seems like this function is longer being used in this file, so I could move it to `buddy_list` -- though I want to look into why it's no longer being used for the navbar tooltip, since the idea was to have similar behavior.

Reply:
I think the tooltip now just shows the actual number, so that function was removed as unused. 

If you need it for the buddy list, you should add a new copy where you'll use it.

Reply:
Moved it